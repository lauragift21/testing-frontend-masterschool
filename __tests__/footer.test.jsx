import { render, screen } from "@testing-library/react";
import Footer from "../src/app/components/footer";

describe("Footer", () => {
  it("renders the Footer component without crashing", () => {
    render(<Footer />);
    expect(screen.getByText("All rights reserved Â©")).toBeInTheDocument();
  });

  it("create a snapshot", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
