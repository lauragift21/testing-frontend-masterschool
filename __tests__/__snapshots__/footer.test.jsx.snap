// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer create a snapshot 1`] = `
<DocumentFragment>
  <footer
    class="text-md py-2"
  >
    <span
      class="float-left"
    >
      2023 Y.
    </span>
    <span
      class="float-right"
    >
       All rights reserved Â©
    </span>
  </footer>
</DocumentFragment>
`;
